pkg/server/test/property_value_history_test.go

```diff
@@ -18,9 +23,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func Test_getPropertyValueHistoryHappyCaseTable(t *testing.T) {
```

Comment:
Did this test go missing?

Reply:
i replaced this test with `Test_get_property_value_history_with_default_aka_table_response_format` which inlines the inputs and test outputs instead of loading separate json files. let me try to reorder to see if this improves the diffs here.